public/usage-examples/graphics/draw_circle_record/draw_circle_record-1-simple.cpp

```diff
@@ -0,0 +1,31 @@
+#include "splashkit.h"
+
+int main()
+{
+    open_window("Draw Circles", 800, 600);
+
+    clear_screen();
+
+    // Define the center of the circles
```

Comment:
This comment line is different, in both C# files you do -
`// Define the center point for all circles`

But in python and cpp this is a different comment. Please update to ensure the comment is the same over all 4 files.